frontend/src/styles/theme.ts

```diff
@@ -51,10 +51,11 @@ const defaultTheme = createTheme({
       letterSpacing: 0.25,
     },
     h3: {
-      fontWeight: 400,
```

Comment:
Can you edit a different heading? This heading is used in the homepage.

Reply:
I don't think were using h4 yet

Reply:
h3 was modified so that it also modifies the homepage, as it will take the same look as the menu in the order worflow.

Reply:
h4 is used in the menu, let me see what i can do

Reply:
Since that version is only used there and h4 is used in multiple places, I went ahead and modified the one in the homepage. Let me know what you think